### PR TITLE
Fix blasi select measurements

### PR DESCRIPTION
**Pull request by dilpath**
:eyes:

### CODE REVIEW

File: Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/measurements.tsv [hunk 1]
```diff
@@ -1,16 +1,253 @@
...
+observable_x_k05k08	condition	inf	0.0060135768
+observable_x_k05k08	condition	inf	0.0073291191
+observable_x_k05k08	condition	inf	0.0063897149
+observable_x_k05k08	condition	inf	0.0065985939
+observable_x_k05k08	condition	inf	0.0066172451
+observable_x_k05k08	condition	inf	0.0063324372
+observable_x_k05k08	condition	inf	0.006899493
+observable_x_k05k08	condition	inf	0.0038748389
 observable_x_k05k08	condition	inf	0.0037448182
-observable_x_k05k12	condition	inf	0.0388013075
```
Comment (elbaraim) [inline Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/measurements.tsv:8]: @dilpath why some of the measurements are removed? could you clarify?

Comment (dilpath) [reply Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/measurements.tsv:8]: They are not removed, just appear at different positions. The old measurements were accidentally the "last" measurement for each observable in the new measurements table.

The new measurements table now has the measurements that were in the original problem.

Comment (dilpath) [reply Benchmark-Models/Blasi_CellSystems2016/petab_select/output/petab/measurements.tsv:8]: e.g.

https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/pull/158/files#diff-499b12588c267d39114860aa60bfaef8c97d003e857d6c0b5dd6539c8d667b03L16

https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/pull/158/files#diff-499b12588c267d39114860aa60bfaef8c97d003e857d6c0b5dd6539c8d667b03R37

**Review by elbaraim — APPROVED** [review]
👍🏽

**Pull request merged**